src/bittrex-client.js

```diff
@@ -129,14 +131,17 @@ class BittrexClient {
    * @param {String|Number} options.price
    * @return {Promise}
    */
-  async sellLimit(market, { quantity, rate } = {}) {
+  async sellLimit(market, { quantity, rate, timeInForce } = {}) {
```

Comment:
To prevent from from a breaking change please set the default:


```suggestion
  async sellLimit(market, { quantity, rate, timeInForce = 'GTC' } = {}) {
```